pgo/cmd/create.go

```diff
@@ -252,6 +253,9 @@ func init() {
 	createClusterCmd.Flags().IntVarP(&Series, "series", "e", 1, "The number of clusters to create in a series.")
 	createClusterCmd.Flags().IntVarP(&ClusterReplicaCount, "replica-count", "", 0, "The number of replicas to create as part of the cluster.")
 	createClusterCmd.Flags().StringVarP(&ContainerResources, "resources-config", "r", "", "The name of a container resource configuration in pgo.yaml that holds CPU and memory requests and limits.")
+	createClusterCmd.Flags().StringVarP(&PodAntiAffinity, "pod-anti-affinity", "", "",
+		"Specifies the type of anti-affinity that should be utilized when applying  "+
+			"default pod anti-affinity rules to PG clusters (default \"preferred\")")
```

Comment:
The default value is `""` which, if I understand correctly, defers to pgo.yaml which then defaults to `preferred`. Should we refrain from mentioning "default" here?

Reply:
If no value is provided for `pod-anti-affinity`, then the default value of `preferred` will be applied when creating the cluster.  I therefore mentioned the default as being `preferred` here, so that the user will understand that `preferred` will the default form of anti-affinity applied to their cluster once created.